src/content/protocols/pancakeswap-v2-bsc.md

```diff
@@ -0,0 +1,150 @@
+
+## Permission
+
+| Contract       | Function          | Impact                                                                                                                                                                                                                                                                                    | Owner / Signer                  |
+| -------------- | ----------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------- |
+| PancakeFactory | setFeeTo          | Sets a new `feeTo` address that receives the 0.08 % protocol cut from **every** pool. This instantly diverts the entire fee stream.                                                                                                                                                       | Multisig `0xcEba6028…a0e`       |
+| PancakeFactory | setFeeToSetter    | Assigns the right to call `setFeeTo` to a new address. This hands permanent control of fee-switch power to a different signer set. A malicious transfer could move control from a multisig to a single EOA, allowing unilateral fee theft.                                                | Multisig `0xcEba6028…a0e`       |
+| MasterChefV2   | add               | Creates a new farm with arbitrary allocation points. This reshapes CAKE distribution and can siphon rewards from existing pools. A malicious owner could add a “ghost” pool with zero real liquidity that captures 100 % of emissions.                                                    | Timelock `0xA1f482Dc…e8fE4`     |
+| MasterChefV2   | set               | Rewrites `allocPoint` of any existing pool. This re-weights how CAKE emissions are shared. A hostile change could drop public pools to near-zero rewards and funnel emissions to insider pools.                                                                                           | Timelock `0xA1f482Dc…e8fE4`     |
+| MasterChefV2   | updateCakeRate    | Updates the percentages sent to burn, regular, and special farms. This alters CAKE’s inflation schedule. Setting burn rate to 0 % while doubling farm rewards would inflate supply rapidly and erode token value.                                                                         | Timelock `0xA1f482Dc…e8fE4`     |
+| MasterChefV2   | burnCake          | Transfers accumulated CAKE to the `burnAdmin` address. Intended to reduce supply, it actually sends tokens to whatever address is set. If that address is switched to an attacker wallet, “burns” become free payouts.                                                                    | Timelock `0xA1f482Dc…e8fE4`     |
```

Comment:
Would be good to mention the burn recipient is actually one of the multisigs at the moment